Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/EditorVertexSelection.h

```diff
@@ -291,16 +300,20 @@ namespace AzToolsFramework
     //! EditorVertexSelectionFixed provides selection and editing for a fixed length number of
     //! vertices. New vertices cannot be inserted/added or removed.
     template<typename Vertex>
-    class EditorVertexSelectionFixed : public EditorVertexSelectionBase<Vertex>
+    class EditorVertexSelectionFixed
+        : public EditorVertexSelectionBase<Vertex>
+        , private AzToolsFramework::EditorVertexSelectionRequestBus::Handler
     {
     public:
         AZ_CLASS_ALLOCATOR_DECL
 
-        EditorVertexSelectionFixed() = default;
-        EditorVertexSelectionFixed(EditorVertexSelectionFixed&&) = default;
-        EditorVertexSelectionFixed& operator=(EditorVertexSelectionFixed&&) = default;
+        EditorVertexSelectionFixed();
+        ~EditorVertexSelectionFixed();
 
     private:
+        // EditorVertexSelectionRequestBus overrides ...
+        virtual void ClearVertexSelection() override;
```

Comment:
update: We shouldn't add these here - `EditorVertexSelectionFixed` explicitly does not implement this bus as you cannot change the number of vertices that are stored. This should only be implemented in `EditorVertexSelectionVariable`